package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "monext",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./src/server.js",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/GabrielNogueiraBR/Monext.git"
+  },
+  
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/GabrielNogueiraBR/Monext/issues"
+  },
+  "homepage": "https://github.com/GabrielNogueiraBR/Monext#readme",
+  "scripts": {
+    "dev": "nodemon -e 'js,html,css' './server.js'",
+    "lint": "eslint ./src",
+    "lint:fix": "eslint --fix . --ignore-pattern './node_modules'",
```

Comment:
Esse script `npm run lint:fix` vai fazer a correção de Lint que for necessária. (não é preciso rodar o script acima depois esse, pode rodar direto esse aqui)